guides/v15-mdc-migration.md

```diff
@@ -0,0 +1,217 @@
+The components in the following packages have been rewritten:
+* @angular/material/autocomplete
+* @angular/material/button
+* @angular/material/card
+* @angular/material/checkbox
+* @angular/material/chips
+* @angular/material/core
+* @angular/material/dialog
+* @angular/material/form-field
+* @angular/material/input
+* @angular/material/list
+* @angular/material/menu
+* @angular/material/paginator
+* @angular/material/progress-bar
+* @angular/material/progress-spinner
+* @angular/material/radio
+* @angular/material/select
+* @angular/material/slide-toggle
+* @angular/material/slider
+* @angular/material/snack-bar
+* @angular/material/table
+* @angular/material/tabs
+* @angular/material/tooltip
+
+The rewritten components offer several benefits over the old implementations, including:
+* Improved accessibility
+* Better adherence to the Material Design spec
+* Faster adoption of future versions of the Material Design spec, due to being based on common 
+  infrastructure
+
+## What has changed?
+
+The new components have different internal DOM and CSS styles. However, the TypeScript APIs and
+component/directive selectors for the new components have been kept as close as possible to the old
+implementation. This makes it straightforward to migrate your app and get it running with the new
+components.
+
+Due to the new DOM and CSS, you will likely find that some styles in your app need to be adjusted,
+particularly if your CSS is overriding styles on internal elements on any of the migrated
+components.
+
+There are a few components with larger changes to their APIs that were necessary in order to
+integrate with MDC. These components include:
+* form-field
+* chips
+* slider
+* list
+
+See below for a [detailed list of changes](#Detailed List of Changes) to be aware of for these and
+other components.
+
+The old implementation of each new component is now deprecated, but still available under a "legacy"
+package. e.g. The old `mat-button` implementation can be used by importing the legacy button module.
+
+```ts
+import {MatLegacyButtonModule} from '@angular/material/legacy-button';
+```
+
+## How to Migrate
+
+Migrating an existing app to use MDC based components is a tool-assisted migration. The Angular
+Material team has built schematics to handle as much of the work as possible. However, some manual
+effort will be needed.
+
+You can reduce the amount of manual effort needed by ensuring that your app follows good practices
+before migrating.
+* Avoid overriding styles on internal Angular Material elements in your CSS as much as possible. If
+  you find yourself frequently overriding styles on internal elements, consider using a component
+  that is designed for more style customization, such as the ones available in the
+  [Angular CDK](/cdk).
+* Use [component harnesses](./using-component-harnesses) to interact with Angular Material
+  components in tests rather than probing internal elements, properties, or methods of the
```

Comment:
I would use the word targeting instead of probing here to make it more clear

Reply:
"targeting" makes sense for CSS selectors, I'm not sure if it makes sense for properties and methods, but I'm open to different word choice, I'm not a languager

Reply:
changed to "inspecting" which is hopefully clearer